srv/components/ha/cortx-ha/init.sls

```diff
@@ -19,7 +19,7 @@
 include:
   - components.ha.cortx-ha.prepare
   - components.ha.cortx-ha.install
-  - components.ha.cortx-ha.config
+  - components.ha.cortx-ha.config.init_mod
```

Comment:
Should be `components.ha.cortx-ha.config`, no?
Also, where is `srv/components/ha/cortx-ha/config/init.sls` that includes the 3 config sub-modules?
We need it until we make the switch to CES workflow

Reply:
Inside the component's overall init.sls , we're calling config.init_mod.sls , which in turn, handles all config phases within. There's no more config.sls file, its deleted and we have only config/ folder inside the components.